automation/vars/default.yaml

```diff
@@ -49,7 +49,7 @@ vas:
         values:
           - name: service-values
             src_file: service-values.yaml
-          - name: edpm-values-post-ceph
+          - name: edpm-nodeset-values-post-ceph
```

Comment:
isn't it a deployment, meaning `edpm-deployment-values-post-ceph` instead?

Reply:
actually there are some changes to nodeset, it add some ceph related variables:
https://github.com/fao89/architecture/blob/edpm_values/examples/va/hci/nodeset-values.yaml#L18-L23

Reply:
hmmm in the framework, those data are set from the ceph.yml play/hook. but OK, let's keep it here. We don't edit it anyway from the framework afaik.